.travis.yml

```diff
@@ -3,7 +3,7 @@ rust:
   - stable
   - beta
   - nightly
-  - 1.43.1
+  - 1.45.0
```

Comment:
The minimum supported version of `tokio` is 1.45.
https://github.com/tokio-rs/tokio#supported-rust-versions